README.md

```diff
@@ -290,6 +290,25 @@ Create a custom configuration for Developer Hub called `app-config-rhdh` by crea
 
 > IT IS INCREDIBLY IMPORTANT THAT THE URLS IN THIS CONFIGURATION ARE CORRECT!!!
 
+NOTE for BUG [RHIDP-3217](https://issues.redhat.com/browse/RHIDP-3217?focusedId=25156525&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-25156525):
```

Comment:
This is honestly probably not the best place to keep this information.  I fully expect a lot of this readme to change in the coming months as we refine dynamic plugin packaging.